examples/printerdemo_mcu/zephyr/README.md

```diff
@@ -0,0 +1,90 @@
+<!-- Copyright © SixtyFPS GmbH <info@slint.dev> ; SPDX-License-Identifier: MIT -->
+
+# Printer Demo with Zephyr
```

Comment:
For expectation setting, I think that it would make sense to mention some of the known issues and limitations here, the ones you also mention in the PR description itself. Later I think when we link to the README, it's good to get an idea what's there and what isn't without having to look for the PR :)

Reply:
Done. I simplified them or changed the language slightly to be more appropriate for documentation.